packages/dart_firebase_admin/lib/src/auth/token_verifier.dart

```diff
@@ -249,6 +247,17 @@ class TokenProvider {
     required this.tenant,
   });
 
+  @internal
+  factory TokenProvider.fromMap(Map<dynamic, dynamic> map) {
+    return TokenProvider(
+      identities: map['identities']! as Map<String, Object?>,
```

Comment:
Since behavior changed here, would you mind writing a test?

Reply:
Done.